File: kotlinames-common/src/main/java/com/nagopy/android/kotlinames/property/nullable/KNullableBytePropertyName.java
package com.nagopy.android.kotlinames.property.nullable;

import com.nagopy.android.kotlinames.KNullablePropertyName;
import com.nagopy.android.kotlinames.KSortablePropertyName;

public class KNullableBytePropertyName implements KNullablePropertyName<Byte>, KSortablePropertyName<Byte> {

    private final String name;

    public KNullableBytePropertyName(String name) {
        this.name = name;
    }

    @Override
    public String name() {
        return name;
    }
}

File: kotlinames/src/main/kotlin/com/nagopy/android/kotlinames/property/nullable/KNullableBytePropertyName.kt
package com.nagopy.android.kotlinames.property.nullable

import com.nagopy.android.kotlinames.KNullablePropertyName
import com.nagopy.android.kotlinames.KSortablePropertyName

class KNullableBytePropertyName(val name: String) : KNullablePropertyName<Byte>, KSortablePropertyName<Byte> {

    override fun name(): String = name

}
